includes/license.php

```diff
@@ -69,8 +69,12 @@ function pmpro_license_isValid($key = NULL, $type = NULL, $force = false) {
 	}
 	
 	// Check if a specific type.
-	if ( ! empty( $type ) && $type != $pmpro_license_check['license'] ) {
-		return false;
+	if ( ! empty( $type ) ) {
+		$type = (array)$type;
+		
+		if ( ! in_array( $pmpro_license_check['license'], $type ) ) {

```

Comment:
Best practices for WP PHPCS is to use third param as `true`

```suggestion
		if ( ! in_array( $pmpro_license_check['license'], $type, true ) ) {

```